examples/ecs-firelens/src/main/java/App.java

```diff
@@ -75,6 +77,7 @@ public void handle(HttpExchange he) throws IOException {
             logger.putProperty("Url", he.getRequestURI());
             try {
                 logger.putMetric("ProcessingTime", System.currentTimeMillis() - time, Unit.MILLISECONDS);
+                logger.putMetric("ProcessingTimeInHighResolution", System.currentTimeMillis() - time, Unit.MILLISECONDS, StorageResolution.HIGH);
```

Comment:
Since this is an example, we should probably only have 1 metric for processing time since realistically a customer won't have 2 identical metrics.

Reply:
Thanks for pointing this out. Updated the example!